### DISCUSSION TITLE
Wohnraumbelüftung Steuern (Shelly)

### DISCUSSION THREAD

**Discussion by dennisbausch** [camueller/SmartApplianceEnabler#275] [Allgemein]
Hallo zusammen , 

ich habe bis jetzt immer nur Geräte gemessen und sie an das Portal übergeben , nun will ich mich auch an die Steuerung eines Gerätes wagen hab aber irgendwie Startschwierigkeiten.

Ich habe eine Wohnraumlüftung die wie folgt steuern möchte. 

Wenn PV Überschuss vorhanden dann soll sie laufen. Schalter ein 

Wenn kein PV Überschuss vorhanden soll die Lüftung sich ausschalten aber nur für 2 Stunde und dann für mindestens 1 Stunde wieder laufen. 

Wie muss ich das mit den vorhandenen Einstellungsmöglichkeiten einstellen? 

Danke  

Perspektivisch stelle ich mir vor die einzelnen Lüfterstufen je nach Überschuss anzusteuern bis runter auf einen Mindestlüftung (Aktuell Steuer ich die Lüftungsanlage über eine KNX Logik und eine Wago die verschiedene Stromstärken ausgibt.)

Hat schon jemand seine Lüftungsanlage gesteuert ?

**Comment by Trunks1982** [comment]
Für mich macht die Wohnraumlüftungssteuerung über pv überschuss keinen sinn.Sollte eher nach Feuchte steuern.

**Comment by camueller** [comment]
> Wenn kein PV Überschuss vorhanden soll die Lüftung sich ausschalten aber nur für 2 Stunde und dann für mindestens 1 Stunde wieder laufen.

Das lässt sich so nicht abbilden. Und es macht auch keinen Sinn, genausowenig, wie beim Kühlschrank oder der Kühltruhe :-)

**Comment by dennisbausch** [comment]
Die KWL kann von der Leistung eingestellt werden. 10-100% .  ( Die Werkssteuerung ist nicht gerade intelligent und erhöht nur die eingestellte Leistung um 20% wenn am Bedienteil die Luftfeuchtigkeit zu hoch ist.)     

Ich habe die Steuerung über eine Logik in meinem Gira Homeserver laufen die die eingestellte Leistung stufenweise erhöht wenn die Luftfeuchtigkeit und oder der CO2 Gehalt an meinen Netatmo Sensoren zu hoch wird. Ebenfalls wird wenn geduscht oder gekocht wird die Leistung für eine gewisse zeit erhöht erhöht.  Geplant ist noch die Außentemperatur zu berücksichtigen um die Leistung zu Drossel oder zu erhöhen.  

Es macht aus meiner Sicht schon Sinn die PV Leistung ebenfalls berücksichtigen und die KWL von der Leistung runterzufahren  wenn nicht genug  PV Überschuss da ist.  

Die KWL hat bei ca. 50 % Leistung 50 Watt bei 100 %ca. 110 Watt

Für die normale Lüftung lauft die Anlage auf 60 % 

Abhängig von PV Überschuss und Akku Stand könnte ich bei verschiedenen Szenerien die Leistung anders ansteuern .

Beispiele

Sommer  = Aussentemperatur  niedriger als Innentemperatur Akku reicht noch bis Sonnenaufgang dann Leistung 15 % höher 
CO2 und Luftfeuchtigkeit  OK Leistung  - 15 Prozent  wenn Akku nicht ausreichend um über die Nacht zu kommen 
Lüftung aus für maximal 2 Stunden wenn alle anderen werte OK und PV keine Leistung     




@Trunks1982 @camueller 

Bei Kühlschränken und Kühltruhen macht es wie ihr geschrieben habt keinen sin.
